python/createCollectionDownload.py

```diff
@@ -2,75 +2,189 @@
 import requests
 import json
 import time
+from datetime import datetime
 
-def check_api_key(cxy_api_key):
+def request_api(url:str, headers:str, method='GET', params=None, data={}):
```

Comment:
I'm not sure if it makes sense to just move the 

```
    headers = {
        'x-apikey': cxy_api_key,
        'x-Application': 'Python API Call',
        'content-type': 'application/json'
        }
```
from donwload_collection to this method, and instead accept cxy_api_key as input? I don't see why we'd want to mess around with headers outside of the method calling the api

Reply:
True. I was going back and forth between the headers of the API key but you're right, it makes more sense to have it in the method that deals with the requests. Resolved!